modules/gapi/include/opencv2/gapi/streaming.hpp

```diff
@@ -0,0 +1,59 @@
+//
+// Copyright 2020 Intel Corporation.
+//
+// This software and the related documents are Intel copyrighted materials,
+// and your use of them is governed by the express license under which they
+// were provided to you ("License"). Unless the License provides otherwise,
+// you may not use, modify, copy, publish, distribute, disclose or transmit
+// this software or the related documents without Intel's prior written
+// permission.
+//
+// This software and the related documents are provided as is, with no
+// express or implied warranties, other than those that are expressly
+// stated in the License.
+//
+
+#ifndef OPENCV_GAPI_STREAMING_STREAMING_HPP
+#define OPENCV_GAPI_STREAMING_STREAMING_HPP
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+
+namespace cv {
+namespace gapi {
+namespace streaming {
+
+GAPI_EXPORTS cv::gapi::GBackend backend();
+
+G_API_OP(GCopy, <GFrame(GFrame)>, "com.intel.streaming.copy")
```

Comment:
`org.opencv.` We don't have anything with `com.intel`. If we have, that needs to be cleared out!!!

Reply:
Fixed. Didn't found this name anywhere else